lighthouse-cli/test/smokehouse/dobetterweb/dbw-expectations.js

```diff
@@ -35,13 +35,13 @@ module.exports = [
         extendedInfo: {
           value: {
             results: {
-              length: 18,
+              length: 19,
```

Comment:
had to bump these as I added another request to the dbw tester file

Reply:
shall we finally just convert this to `'>15'` or something?